media/css/sandstone/lib.less

```diff
@@ -52,7 +53,7 @@
 @linkBlueHover:      rgb(0,83,159);
 @linkSkyBlue:        rgb(103,167,208);
 @linkSpaceBlue:      rgb(47,138,202);
-@linkSpacePurple:    rgb(147,111,188);
+@linkSpaceOrange:    rgb(255,173,31);
```

Comment:
Agree. The headings should probably remain while, while the links could be `@linkSpaceBlue`.